Source/RapyutaSimulationPlugins/Private/Core/RRROS2Utils.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
```

Comment:
Do you need this class? 
isn't is simple to add constructor with same argument to ROS2 Publisher?

Reply:
Ah indeed, this could be moved UROS2Publisher class